lightbluetent/api.py

```diff
@@ -124,59 +170,45 @@ def create(self, params):
         message = ""
         response = self.request("create", params)
 
-        if "returncode" not in response or "message" not in response:
+        if response is None:
+            created = False
+            message = "Error: Malformed response from server."
+            current_app.logger.info("create: malformed response from server")
+        elif "returncode" not in response or "message" not in response:
             created = False
             message = "Error: Malformed response from server."
+            current_app.logger.info("create: malformed response from server")
         else:
             if response["returncode"] != "SUCCESS":
                 created = False
                 message = response["message"]
+                current_app.logger.info(f"create: error: { response.message }")
             else:
                 created = True
 
         return (created, message)
 
 
-    # Join a call.
-    # Returns (True, "[call_url]") if successful, otherwise
-    # (False, "[error_message]").
-    def join(self, params):
-        joined = False
-        message = ""
-        call_url = ""
-        response = self.request("join", params)
-
-        if "returncode" not in response or "message" not in response:
-            joined = False
-            message = "Error: Malformed response from server."
-        else:
-            message = response["message"]
-            if response["returncode"] != "SUCCESS":
-                joined = False
-            else:
-                joined = True
-                call_url = response["url"]
-
-        if joined:
-            return (joined, call_url)
-        else:
-            return (joined, message)
-
-
     # End a meeting.
     # Returns (True, "") if successful, otherwise (False, "[error_message]").
     def end(self, params):
         ended = False
         message = ""
         response = self.request("end", params)
 
-        if "returncode" not in response or "message" not in response:
+        if response is None:
+            ended = False
+            message = "Error: Malformed response from server."
+            current_app.logger.info(f"end: malformed response from server")
+        elif "returncode" not in response or "message" not in response:
             ended = False
             message = "Error: Malformed response from server."
+            current_app.logger.info("end: malformed response from server")
         else:
             if response["returncode"] != "SUCCESS":
                 ended = False
                 message = response["message"]
+                current_app.logger.info(f"end: error: { response.message }")
```

Comment:
I suggest we make better use of log level granularity: info, warning, error, critical. logger.error is probably best. Perhaps some other times we log could benefit from this?

Reply:
Yes, I'll go through and adjust this. Should probably be done in a separate PR though.

Reply:
Agree.